testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -65,26 +65,16 @@ public void testColumnMappingModeNone()
     @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
     public void testColumnMappingModeId()
     {
```

Comment:
https://docs.databricks.com/delta/delta-column-mapping.html

>  It also allows users to name Delta table columns by using characters that are not allowed by Parquet, such as spaces, so that users can directly ingest CSV or JSON data into Delta without the need to rename columns due to previous character constraints.

Let's add a test case showcasing this functionality.

https://docs.databricks.com/delta/delta-column-mapping.html#supported-characters-in-column-names

> When column mapping is enabled for a Delta table, you can include spaces as well as any of these characters in the table’s column names: ,;{}()\n\t=.